appinfo/info.xml

```diff
@@ -5,7 +5,7 @@
 	<name>End-to-End Encryption</name>
 	<summary>End-to-end encryption endpoint</summary>
 	<description>Provides the necessary endpoint to enable end-to-end encryption.</description>
-	<version>1.10.1</version>
+	<version>1.10.0</version>
```

Comment:
Decreasing version number?
What's going on?

Reply:
https://apps.nextcloud.com/apps/end_to_end_encryption 1.10.0 was never released, I suppose 1.9.1 was transformed to 1.10.1 instead of 1.10.0 when branched.